R/db2.R

```diff
@@ -429,16 +429,13 @@ report_data_import <- function(con, name, id, config) {
 
   if (!is.null(dat_rds$meta$workflow)) {
     sql_batch <- "SELECT id FROM workflow WHERE id = $1"
-    if (nrow(DBI::dbGetQuery(con, sql_batch, dat_rds$meta$workflow$id)) == 0L) {
-      workflow <- data_frame(
-        id = dat_rds$meta$workflow$id,
-        name = dat_rds$meta$workflow$name
-      )
+    if (nrow(DBI::dbGetQuery(con, sql_batch, dat_rds$meta$workflow)) == 0L) {
+      workflow <- data_frame(id = dat_rds$meta$workflow)
       DBI::dbWriteTable(con, "workflow", workflow, append = TRUE)
     }
     report_version_workflow <- data_frame(
       report_version = id,
-      workflow_id = dat_rds$meta$workflow$id
+      workflow_id = dat_rds$meta$workflow
```

Comment:
Does this mean we need a metadata migration?

Reply:
Discussed this, no one has used it so we can get away with not writing a migration